pkg/cloudprovider/providers/openstack/openstack_loadbalancer.go

```diff
@@ -49,6 +50,18 @@ import (
 const loadbalancerActiveTimeoutSeconds = 120
 const loadbalancerDeleteTimeoutSeconds = 30
 
+// svcAnnotationProvider and svcAnnotationSubnetID are service annotations that override their corresponding default (i.e. from cloud-config) settings
```

Comment:
This block will now become the canonical place where these options are documented - please add comments to each describing expected values, behaviour that the option affects, etc.  (Only has to be a line or two, see the AWS cloud provider for examples)